backend/app/tests/utils.py

```diff
@@ -160,6 +163,33 @@ def upload_file(
     assert response.json().get("id") is not None
     return response.json()
 
+def upload_files(
+    client: TestClient,
+    headers: dict,
+    dataset_id: str,
+    filenames=[filename_example_1, filename_example_2],
+    file_contents=[file_content_example_1, file_content_example_2]
+):
+    """Uploads a dummy file (optionally with custom name/content) to a dataset and returns the JSON."""
+    upload_files = []
+    uploads = []
+    for i in range(0, len(filenames)):
+        with open(filenames[i], "w") as tempf:
+            tempf.write(file_contents[i])
+            uploads.append(open(filenames[i], "rb"))
```

Comment:
i think you just want filenames[i] in the list if you are calling os.remove() later

Reply:
Should be fixed now.